src/SDKs/EventHub/EventHub.Tests/Tests/ScenarioTests.ConsumergroupsTests.CRUD.cs

```diff
@@ -1,115 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-
-namespace EventHub.Tests.ScenarioTests
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Net;
-    using Microsoft.Azure.Management.EventHub;
-    using Microsoft.Azure.Management.EventHub.Models;
-    using Microsoft.Rest.Azure;
-    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
-    using TestHelper;
-    using Xunit;
-    public partial class ScenarioTests
-    {
-        [Fact]
-        public void ConsumerGroupsCreateGetUpdateDelete()
-        {
-            using (MockContext context = MockContext.Start(this.GetType().FullName))
```

Comment:
Is there a reason you'd like to delete these tests? Are these operations no longer being supported? You can always skip the test if this temporary

Reply:
right now, Preview API do not support the operations other than the tests included. so deleted the test. once the Preview and Stable API operations are merged will restore the tests.

Reply:
In that case, either create a test playlist or skip tests that you wish to restore in the future.